client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -172,6 +185,10 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow, stepName } ) => {
 		enabled: isPreviewingDesign && selectedDesignHasStyleVariations,
 	} );
 
+	if ( isDesignFirstFlow && selectedDesignDetails?.style_variations ) {
```

Comment:
Here we avoid showing them on the lateral menu:

![image](https://github.com/Automattic/wp-calypso/assets/1044309/e21ce3f9-58bc-45b2-a012-e56cf716f7ec)